os/trusted-computing-hardware-requirements.md

```diff
@@ -0,0 +1,58 @@
+# Checking hardware and firmware support for CoreOS Trusted Computing
+
+Trusted Computing requires support in both system hardware and firmware. This document specifies the required support and explains how to determine if a physical machine has the features needed to enable Trusted Computing in CoreOS.
+
+## 1. Check for Trusted Platform Module
+
+Trusted Computing depends on the presence of a Trusted Platform Module, commonly referred to as a TPM. The TPM is a motherboard component responsible for storing the state of the system boot process, and providing a secure communication channel over which this state can be verified. To check for the presence of a TPM, install the latest Alpha version of CoreOS and try to list the TPM device file in the `/sys` system control filesystem:
+
+`# ls /sys/class/tpm/tpm0`
+
+If this returns an error, your system either does not have a TPM, or it is not enabled in the system firmware. Firmware configuration varies by system. Consult your vendor documentation for details.
+
+## 2. Check TPM version
+
+Only version 1.2 TPMs are supported at present. Read the TPM device ID file to discover the TPM version:
+
+`# cat /sys/class/tpm/tpm0/device/id`
+
+If this prints `TRUESTRING`, then the system has a version 1.2 TPM and supports Trusted Computing.
+
+If this prints `MSFT0101`, then the system has a version 2.0 TPM. Support for version 2.0 TPMs will be added in future CoreOS releases.
+
+## 3. Check TPM is enabled and active
+
+The TPM device provides control files in the `/sys` filesystem, as seen above. Read the `enabled` and `active` files to check TPM status:
+
+```sh
+# cat /sys/class/tpm/tpm0/device/enabled
+# cat /sys/class/tpm/tpm0/device/active
+```
+
+If either of these commands prints "0", reconfigure the TPM by writing a code for TPM activation at the next system boot to the PPI `request` file:
+
+`# echo 6 > /sys/class/tpm/tpm0/device/ppi/request`
+
+Reboot the system and check TPM status again, as in Step 3.
+
+## 4. Ensure the boot process is measured
+
+The CoreOS bootloader will record the state of boot components during the
+boot process &mdash; *measuring* each part, in TPM parlance. Verify that this measurement has been successful by reading the TPM device's `pcrs` file, which textually represents the contents of TPM registers:
+
+`# cat /sys/class/tpm/tpm0/device/pcrs`
+
+PCRs 9, 10, 11, 12 and 13, where boot component measurements are recorded, should all contain meaningful values; that is, values that are neither `0`, e.g.:
```

Comment:
I found this hard to read. What about:

> Boot component measurements are recorded PCRs 9, 10, 11, 12 and 13. Each PCR should contain meaningful values instead of all zeros or all max values: